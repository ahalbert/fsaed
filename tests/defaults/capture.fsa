1: /beep/ ->
2: {capture mycapture -> }
3: do s/THIS.IS.CAPTURED/CAPTURED/ mycapture
3: /buzz/ -> 
4: print mycapture
