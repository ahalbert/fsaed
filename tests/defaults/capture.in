beep
THIS IS CAPTURED
boop
foo
buzz
bar
baz
