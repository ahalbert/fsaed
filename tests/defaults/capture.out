beep
boop
foo
buzz
CAPTURED
bar
CAPTURED
baz
